src/scripts/install_k8s.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -ex
+
+if [[ -n "$WITH_K8S" ]]; then
+  # download kind
+curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.8.1/kind-$(uname)-amd64"
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
+
+chmod +x ./kind
+chmod +x ./kubectl
+
+sudo mv ./kubectl /usr/local/bin/kubectl
+
+# create cluster
+kubectl version --client
+./kind create cluster
+
+cat <<EOF > pv.yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pv-hostpath
+  labels:
+    type: host
+spec:
+  persistentVolumeReclaimPolicy: Recycle
+  storageClassName: manual
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/tmp"
+EOF
+
+kubectl apply -f pv.yaml 
```

Comment:
this could be done by popper, or do you see anything that wouldn't work if it did?

Reply:
Users can have a PV created from before that they would like to use which might contain some data that the workflow might be using.
This is particularly important in a shared cluster where each user might have been allocated some PV by the admin. 
Also, since everything created by popper is ephemeral but PVs are not, i felt lets keep PV and popper separate.

What we can do is we can always ask a PV from user (considering it will be a complex PV like aws ebs, etc.) through config and if not provided popper will create a local PV  and select a node randomly (if some node is not schedulable , popper can try out other nodes) and schedule all pods on that node. What do you think?

Reply:
> What we can do is we can always ask a PV from user (considering it will be a complex PV like aws ebs, etc.) through config and if not provided popper will create a local PV and select a node randomly (if some node is not schedulable , popper can try out other nodes) and schedule all pods on that node. What do you think? Basically I am thinking if we can make local PV the default case instead of an exception.

this sounds good!

Reply:
Ok. I would update the code accordingly